src/main/java/bio/terra/service/BillingService.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.service;
```

Comment:
Seems like this should be a pdao - or in my proposed re-org of the cloud-specific stuff, under that.

Reply:
This is a good huddle convo -- I can see both sides. In my mind billing is not primary data, but a service required to support resource management which in turn is a service required to support primary data management.

Reply:
Yes, it isn't primary. My suggestion was to isolate the cloud services with a general set of interfaces in a `cloud` package and the google implementations in `cloud/google` package. Within that we might want to separate pdao from non-pdao things, or not.

The current `filesystem` package is another instance of broken organization.